src/cli/commands/public-cmds/import-cmd.js

```diff
@@ -179,18 +179,18 @@ export default class Import extends Command {
       dependenciesOutput = componentDependenciesOutput + peerDependenciesOutput;
     }
 
-    if (envDependencies && !R.isEmpty(envDependencies)) {
-      envDependenciesOutput = immutableUnshift(
-        envDependencies.map(envDependency => formatPlainComponentItem(envDependency.component)),
+    if (envComponents && !R.isEmpty(envComponents)) {
+      envComponentsOutput = immutableUnshift(
+        envComponents.map(envDependency => formatPlainComponentItem(envDependency.component)),
         chalk.green('the following component environments were installed')
       ).join('\n');
     }
 
     const getImportOutput = () => {
-      if (dependenciesOutput && !envDependenciesOutput) return dependenciesOutput;
-      if (!dependenciesOutput && envDependenciesOutput) return envDependenciesOutput;
-      if (dependenciesOutput && envDependenciesOutput) {
-        return `${dependenciesOutput}\n\n${envDependenciesOutput}`;
+      if (dependenciesOutput && !envComponentsOutput) return dependenciesOutput;
```

Comment:
it hurts us
![image](https://user-images.githubusercontent.com/1002622/43675781-2ed2ec74-97ee-11e8-88c0-4f3cf936f135.png)

Reply:
:)